### PR TITLE
test

### PR DESCRIPTION
**Pull request by CeciliaSung**

### CODE REVIEW

**Review by scisnero — APPROVED** [review]

File: _posts/0000-01-16-CeciliaSung.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![philosoraptor](https://cloud.githubusercontent.com/assets/16547949/25401001/66d677c6-29c2-11e7-9b12-263e036c236e.jpg)
 {: .center}
 
-Caption here
+test
```
Comment (CeciliaSung) [inline _posts/0000-01-16-CeciliaSung.md]: add comments here

File: _posts/0000-01-16-CeciliaSung.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![philosoraptor](https://cloud.githubusercontent.com/assets/16547949/25401001/66d677c6-29c2-11e7-9b12-263e036c236e.jpg)
 {: .center}
```
Comment (scisnero) [inline _posts/0000-01-16-CeciliaSung.md:8]: Leaving a comment 👍

File: _posts/0000-01-16-CeciliaSung.md [hunk 3]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![philosoraptor](https://cloud.githubusercontent.com/assets/16547949/25401001/66d677c6-29c2-11e7-9b12-263e036c236e.jpg)
```
Comment (CeciliaSung) [inline _posts/0000-01-16-CeciliaSung.md:6]: add another comments

File: _posts/0000-01-16-CeciliaSung.md (hunk 2 above)
Comment (scisnero) [reply _posts/0000-01-16-CeciliaSung.md:8]: testing adding another comment 😄

**Review by scisnero — COMMENTED** [review]
Looks good! 👍 😄  @CeciliaSung

**Pull request merged**